concent_api/concent_api/urls.py

```diff
@@ -13,14 +13,17 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url, include
-from django.contrib   import admin
+from django.conf            import settings
+from django.core.exceptions import ImproperlyConfigured
 
-import core.urls
-import gatekeeper.urls
+from concent_api.constants  import AVAILABLE_CONCENT_FEATURES
 
-urlpatterns = [
-    url(r'^',        include(gatekeeper.urls, namespace = 'gatekeeper')),
-    url(r'^api/v1/', include(core.urls, namespace = 'core')),
-    url(r'^admin/',  admin.site.urls),
-]
+urlpatterns = []
+
+if not hasattr(settings, 'CONCENT_FEATURES'):
+    raise ImproperlyConfigured("CONCENT_FEATURES variable must be defined in django settings.")
+
+assert set(settings.CONCENT_FEATURES) - set(AVAILABLE_CONCENT_FEATURES.keys()) == set()
```

Comment:
A comment above this line would be nice.

Reply:
I've added quick fixup.

Reply:
I have meant the line with assert ;)

Reply:
Okey, after talk to @cameel  we decide to switch asset with if statement, and there was also a bug in for loop to fix ;)